CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * User Operator now uses Kafka Admin API to manage SCRAM-SHA-512 credentials.
   All operations done by the User Operator now use Kafka Admin API and connect directly to Kafka instead of ZooKeeper.
   As a result, the environment variables `STRIMZI_ZOOKEEPER_CONNECT` and `STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS` were removed from the User Operator configuration.
+ * Annotate Cluster Operator resource metrics by a namespace label
```

Comment:
```suggestion
 * Annotate Cluster Operator resource metrics with a namespace label
```
I think we want to use with here, but might be worth asking someone with a better knowledge of the English language than myself!

Reply:
As my English is as good as stepping on the lego, I am pinging @PaulRMellor and @laidan6000 to check.

Reply:
@sknot-rh Yes. I'd change to _with_